api/v1alpha1/dnspolicy_types.go

```diff
@@ -214,14 +212,14 @@ type DNSPolicyList struct {
 // By default, this health check will be applied to each unique DNS A Record for
 // the listeners assigned to the target gateway
 type HealthCheckSpec struct {
-	Endpoint                  string                                    `json:"endpoint,omitempty"`
-	Port                      *int                                      `json:"port,omitempty"`
-	Protocol                  *kuadrantdnsv1alpha1.HealthProtocol       `json:"protocol,omitempty"`
-	FailureThreshold          *int                                      `json:"failureThreshold,omitempty"`
-	AdditionalHeadersRef      *kuadrantdnsv1alpha1.AdditionalHeadersRef `json:"additionalHeadersRef,omitempty"`
-	ExpectedResponses         []int                                     `json:"expectedResponses,omitempty"`
-	AllowInsecureCertificates bool                                      `json:"allowInsecureCertificates,omitempty"`
-	Interval                  *metav1.Duration                          `json:"interval,omitempty"`
+	Endpoint                  string           `json:"endpoint,omitempty"`
+	Port                      *int             `json:"port,omitempty"`
+	Protocol                  *string          `json:"protocol,omitempty"`
+	FailureThreshold          *int             `json:"failureThreshold,omitempty"`
+	AdditionalHeadersRef      *string          `json:"additionalHeadersRef,omitempty"`
```

Comment:
I'm not sure if it matters but we seem to be changing the type here. Would we be better to add the `AdditionalHeadersRef` type to the types here to keep it compatible? tbh this might change considerably as we go, but i believe you can add headers to the route53 health checks so might be something we want to keep.

Reply:
My intention was to have this be the name of a secret with additional headers specified in it.